.changeset/honest-poems-visit.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/plugin-scaffolder-react': minor
```

Comment:
Don't think you'll need this changeset when you move the file into the `TemplateWizardPage` folder of the `scaffolder` plugin.

Reply:
I deleted it